util/common.js

```diff
@@ -0,0 +1,21 @@
+const { flatten } = require("flat");
```

Comment:
Any reason why we're not using `flatten` from `lodash` ?

Reply:
flatten from lodash only works arrays